README.md

```diff
@@ -106,12 +104,12 @@ Each package with integration tests has their own instructions for running:
 - [Babylon.js](packages/amazon-sumerian-hosts-babylon/test/integration_test/README.md)
 - [Three.js](packages/amazon-sumerian-hosts-three/test/integration_test/README.md)
 
-### Examples Folder
-Each of the implementation packages have their own examples attached to them. Each of these files contain enough code to stand up and verify the code is working properly. To verify, open up a local server for the package you are modifying:
+### Examples and Demos
+Each of the integration packages have their own examples or demos attached to them. Each of these files contain enough code to stand up and verify the code is working properly. To verify, open up a local server for the package you are modifying:
 ```
 npm run start-{engine-name}

```

Comment:
The demos won't work unless the user has set up the proper infrastructure in their AWS account and updated the `demo-credentials.js` file with their Cognito ID. For these reasons it seems a little premature to tell the user how to run the demos in the top level README (without having to add a lot more detail.) Maybe it would be better to guide them to `packages/demos-babylon/README.md` for details on how to setup and run the Babylon demos and to `packages/amazon-sumerian-hosts-three/README.md` for the Three demo?

Reply:
Changed to point to the actual README.